cdp_scrapers/tests/legistar_content_parser_test.py

```diff
@@ -66,17 +66,19 @@
             # parser4
             "https://longbeach.granicus.com/MediaPlayer.php?view_id=84&clip_id=13404",
             "longbeach",
-            None,
-            "https://archive-video.granicus.com/longbeach/"
-            + "longbeach_cf37b162-9817-428a-9fcd-b72daca8062a.mp4",
+            "http://longbeach.granicus.com//videos/13404/captions.vtt",
```

Comment:
Something may have changed upstream, and these tests were failing assertion on master. I tried to match the data to pass the tests. @dphoria Let me know if this is alright.

Reply:
I've had to update tests from time to time as well due to the municipalities changing the data. I'll check this out.